performance/SwiftUIPerformanceQuickstart/PerformanceExample/README.md

```diff
@@ -9,8 +9,23 @@ while also allowing you to measure the performance of any piece of code in your
 
 Introduction
 ------------
+The app is centered around image tasks: download, classify, saliency map, and upload. Downloading
```

Comment:
The saliency changes should be in a different PR

Reply:
They were merged with https://github.com/firebase/quickstart-ios/pull/1257. I needed to merge those changes in to fix performance CI (storage rules only work for saliency changes). What should I do on this PR?